### DISCUSSION TITLE
swiper cube

### DISCUSSION THREAD

**Discussion by ghost** [nolimits4web/swiper#5632] [Q&A]
Good day,
I use Swiper on the homepage "www.goetten-art.de" (Typo3 11.5.9) and have integrated a slider on the page "Home".
Initializing the script with:     
       
            <script>
              var swiper = new Swiper(".swiper-container", {
                autoplay: {
                    delay: 5000,
                },
                direction: 'horizontal',
                loop: true,
                navigation: {
                    nextEl: ".swiper-button-next",
                    prevEl: ".swiper-button-prev",
                },
                pagination: {
                    el: ".swiper-pagination",
                },
                effect: 'cube',
              });              
            </script>

You can see that after "Slide 7" it starts again from the beginning, but then two slides appear.

What could be the reason for this?

Regards

opaelmar

Translated with www.DeepL.com/Translator (free version)

**Comment by Arhell** [comment]
Here is an example, try to use what is in the example to check, without the property "direction: 'horizontal'," and others
https://codesandbox.io/s/6x3kre?file=/index.html

**Comment by ghost** [comment]
Oh yes
I was trying out different options and had "effect: 'slide'" before. From that then still direction and others.

So it works fine.

Thanks a lot

opaelmar
